modules/registry-operator-configuration-resource-overview.adoc

```diff
@@ -54,4 +54,12 @@ for the route.
 |`replicas`
 |Replica count for the registry.
 
+|`spec.storage.managementState`
+
+|The Image Registry Operator sets the `spec.storage.managementState` parameter to `Managed` on new installations or upgrades of installer-provisioned infrastructure clusters on AWS or Azure.
+
+* `Managed`: Determines that the Image Registry Operator manages underlying storage. If the Image Registry Operator's `managementState` is set to `Removed`, then the storage is deleted.
```

Comment:
In addition to what is already written it may be useful to add something similar to:

"If managementState is set to Managed the image registry operator will always attempt to apply some default configuration on the underlying storage unit. For example: if Managed the operator will try to enable encryption on the S3 bucket before making the it available to the registry. If you don't want the defaults to be applied on the storage you are providing make sure this is set to Unmanaged"